alvr/server_openvr/cpp/platform/win32/CEncoder.cpp

```diff
@@ -19,10 +19,23 @@ void CEncoder::Initialize(std::shared_ptr<CD3DRender> d3dRender) {
     uint32_t encoderWidth, encoderHeight;
     m_FrameRender->GetEncodingResolution(&encoderWidth, &encoderHeight);
 
+    Exception vplException;
     Exception vceException;
     Exception nvencException;
 #ifdef ALVR_GPL
     Exception swException;
+#endif
+
+    try {
+        Debug("Try to use VideoEncoderVPL.\n");
+        m_videoEncoder = std::make_shared<VideoEncoderVPL>(d3dRender, encoderWidth, encoderHeight);
+        m_videoEncoder->Initialize();
+        return;
+    } catch (Exception e) {
+        vplException = e;
+    }
```

Comment:
I think this should go after the other encoders because it's the least likely path to be hit and it shouldn't matter if the other ones get unsuccessfully checked first (except if you have multiple gpus, but we don't do cross gpu anyways and if you have another gpu ur prolly using that one).
I don't know if vpl can trigger on intel integrated graphics, but if it can, making this last is certainly a good way to prevent that from happening.